src/auth/credential.ts

```diff
@@ -302,12 +299,9 @@ export interface Credential {
 export class RefreshTokenCredential implements Credential {
   private refreshToken_: RefreshToken;
 
-  constructor(refreshTokenPathOrObject: string|Object) {
-    if (typeof refreshTokenPathOrObject === 'string') {
-      this.refreshToken_ = RefreshToken.fromPath(refreshTokenPathOrObject);
-    } else {
-      this.refreshToken_ = new RefreshToken(refreshTokenPathOrObject);
-    }
+  constructor(refreshTokenPathOrObject: string | object) {
+    this.refreshToken_ = (typeof refreshTokenPathOrObject === 'string')
+      ? RefreshToken.fromPath(refreshTokenPathOrObject) : new RefreshToken(refreshTokenPathOrObject);
```

Comment:
Better for readability to have the ? on the previous line

Reply:
Done